src/mahoji/commands/fish.ts

```diff
@@ -146,11 +172,13 @@ export const fishCommand: OSBMahojiCommand = {
 				quantity = maxCanDo;
 			}
 
-			const cost = new Bank();
-			cost.add(baseCost.multiply(quantity));
+			shouldRemoveFromBank = true;
+			cost.add(fish.bait, quantity);
+		}
 
-			// Remove the bait from their bank.
-			await user.removeItemsFromBank(new Bank().add(fish.bait, quantity));
+		if (shouldRemoveFromBank) {
```

Comment:
Unless im misunderstanding, this variable isnt needed, you can just do `if (cost.length > 0)` to check if there any items in the cost bank.

Reply:
You're right, I fixed that.